### PR TITLE
Clean up deprecation warnings from Hopac

### PR DESCRIPTION
**Pull request by neoeinstein**

### CODE REVIEW

File: HttpFs/HttpFs.fs [hunk 1]
```diff
@@ -142,8 +142,8 @@ module Logging =
 
   let NoopLogger =
     { new Logger with
-        member x.logVerbose evaluate = Alt.always (Promise.Now.withValue ())
-        member x.log message = Alt.always (Promise.Now.withValue ())
+        member x.logVerbose evaluate = Alt.always (Promise ())
+        member x.log message = Alt.always (Promise ())
```
Comment (polytypic) [inline HttpFs/HttpFs.fs:146]: That actually has the wrong semantics. :(

See [here](https://github.com/Hopac/Hopac/issues/113).

@haf Yeah, I'm now convinced that we ~~should~~ must have a clearer API for creating promises.

**Comment by neoeinstein** [comment]
And I even saw that issue...

**Comment by haf** [comment]
Will you fix the bug you introduced in this PR or should I continue down #95 ?

**Comment by neoeinstein** [comment]
I can change the `Promise ()` calls to something like `memo (Job.unit ())`, but it may be better for me to just revert those two lines and/or just kill this PR for now.

**Comment by haf** [comment]
Let's do it in a larger PR then.

**Pull request closed without merging**